app/pages/profile.jsx

```diff
@@ -113,7 +119,7 @@ class Profile extends React.Component {
         var profile = {};
         var files = {};
 
-        const inputBirthday = new Date(this.state.birthday);
+        const inputBirthday = + new Date(this.state.birthday);
```

Comment:
This works, dunno what the benefit is of coercing it to the timestamp rather than getting it directly by using `Date.getTime()`